airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -72,9 +72,9 @@
   icon: salesforce.svg
 - sourceDefinitionId: e094cb9a-26de-4645-8761-65c0c425d1de
   name: Stripe
-  dockerRepository: airbyte/source-stripe-singer
-  dockerImageTag: 0.2.1
-  documentationUrl: https://hub.docker.com/r/airbyte/integration-singer-stripe-source
+  dockerRepository: airbyte/source-stripe
+  dockerImageTag: 0.1.7
```

Comment:
latest version is 0.1.8

Reply:
I will publish the new version and update it to 0.1.9 already, because there were changes in the connector code.